db/institutions.yml

```diff
@@ -205,6 +205,16 @@ ucopenhagen:
   name: University of Copenhagen
   state: ''
   zip: ''
+ccny:
+  aka: [City College, CUNY City College]
+  city: New York
+  country: USA
+  departments: {cheme: Department of Chemical Engineering, chemistry: Department of Chemisty and Biochemistry}
+  name: The City College of New York
+  schools: Grove School of Engineering
```

Comment:
I think this should also be a dict, similar format to the departments.

Reply:
for the aka or the department

Reply:
the schools.  Please look at Columbiau as an example.